R/pretty_rmd.R

```diff
@@ -30,6 +32,7 @@ pretty_rmd <- function(input,
                        open_output = TRUE, 
                        create_library = TRUE,
                        chunks = NULL,
+                       lookup_order = c("searchpath", "document", "library", "sos"),
```

Comment:
This isnt being used in the function itself. what is the purpose of it?

Reply:
Hey @yonicd ,
This was a feature that I had intended to build out but never did :-/ 
It was meant to allow the user to specify the priority of where packages are searched for (and omit some if they wish)

Reply:
Ah ok. I thought that that was the intention, but i did see it implemented in the function after :)